int32      field1 # 92
Unmigrated field2 # (12, "uuiasjs", 61.7)
